h/services/bulk_executor/_actions.py

```diff
@@ -124,6 +124,58 @@ def execute(self, batch, effective_user_id=None, **_):
         ]
 
 
+class GroupMembershipCreateAction(DBAction):
+    """Perform a bulk group membership create.
+
+    :raises ConflictingDataError: Upon trying to create a membership to a user
+                                  or group that does not exist
+    """
+
+    def execute(self, batch, on_duplicate="continue", **_):
+        """
+        :param on_duplicate: Specify behavior when a record already exists. The
+                             default is "continue"
+        """
+        if on_duplicate != "continue":
+            raise UnsupportedOperationError(
+                "Create modes other than 'continue' have not been implemented"
+            )
+
+        values = [
+            {"user_id": command.body.member.id, "group_id": command.body.group.id}
+            for command in batch
+        ]
+
+        stmt = insert(GroupMembership).values(values)
+
+        if on_duplicate == "continue":
```

Comment:
Yeah, this is dumb because it's the only thing it can be right now, but if we wanted to fail on duplicates, this is how you'd do it. 

This would raise a conflict error which we would have to catch below.